suites/e2e/suite.js

```diff
@@ -59,51 +74,41 @@ module.exports = {
 						},
 						// persistentLogging is managed by the supervisor and only read at first boot
 						persistentLogging: true,
+						// Set local mode so we can perform local pushes of containers to the DUT
+						localMode: true,
 					},
```

Comment:
Depending on the version of the OS image, we may need to add `developmentMode: true` to the configJson

Reply:
Good point